src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -115,6 +115,35 @@ public void TestMap()
         }
 
         [Test]
+        public void TestCurrency()

```

Comment:
Can you separate this in multiple tests, for example based on the culture?
It is best to have a limited number of asserts in one test.